src/pages/Clients/Eth2/Teku.tsx

```diff
@@ -14,123 +15,174 @@ import { TEKU_INSTALLATION_URL } from '../../../utils/envVars';
-              validators-keys: [{'<'}KEY_DIR{'>'}:{'<'}PASS_DIR{'>'}]
-            </Code>{' '}
-            or
-            <Code className="px5 ml5">
-              {/* eslint-disable-next-line react/no-unescaped-entities */}
-              validators-keys: [{'<'}KEY_FILE_1{'>'}:{'<'}PASS_FILE_1{'>'},...,
-              {'<'}KEY_FILE_N{'>'}:{'<'}PASS_FILE_N{'>'}]
-            </Code>
-            .
+            Teku can also be configured via a YAML file which is passed in via a
+            few different ways.
           </Text>
+          <ul>
+            <li>
+              <Text>The config file CLI argument</Text>
+            </li>
+            <ul>
+              <li>
+                <Code className="px5 ml5">--config-file</Code>
+              </li>
+            </ul>
+            <li>
+              <Text>The Teku config file environment variable</Text>
+            </li>
+            <ul>
+              <li>
+                <Code className="px5 ml5">TEKU_CONFIG_FILE</Code>
+              </li>
+            </ul>
+          </ul>
+          <Text>The YAML files can have different syntaxes.</Text>
+          <ul>
+            <li>
+              <Code className="px5 ml5">
+                {/* eslint-disable-next-line react/no-unescaped-entities */}
+                validators-keys: [{'<'}KEY_DIR{'>'}:{'<'}PASS_DIR{'>'}]
+              </Code>{' '}
+            </li>
+            <li>
+              <Code className="px5 ml5">
+                {/* eslint-disable-next-line react/no-unescaped-entities */}
+                validators-keys: [{'<'}KEY_FILE_1{'>'}:{'<'}PASS_FILE_1{'>'}
+                ,...,
+                {'<'}KEY_FILE_N{'>'}:{'<'}PASS_FILE_N{'>'}]
+              </Code>
+            </li>
+          </ul>
           <Heading level={4} className="mt10 mb5">
-            Note:
+            Note
           </Heading>
           <Text>
-            <Code className="px5 ml5">
-              {'<'}KEY_DIR{'>'}:{'<'}PASS_DIR{'>'}
-            </Code>{' '}
-            will find{' '}
-            <Code className="px5 ml5">
-              {'<'}KEY_DIR{'>/**'}.json
-            </Code>
-            , and expect to find{' '}
-            <Code className="px5 ml5">
-              {'<'}PASS_DIR{'>/**'}.txt
-            </Code>
-            .{' '}
-            <Code className="px5 ml5">
-              {'<'}KEY_FILE{'>'}:{'<'}PASS_FILE{'>'}
-            </Code>{' '}
-            will expect that the file{' '}
-            <Code className="px5 ml5">
-              {'<'}KEY_FILE{'>'}
-            </Code>{' '}
-            exists, and the file containing the password for it is{' '}
-            <Code className="px5 ml5">
-              {'<'}PASS_FILE{'>'}
-            </Code>
-            . The path separator is operating system dependent, and should be{' '}
-            <Code className="px5 ml5">;</Code> in Windows rather than{' '}
-            <Code className="px5 ml5">:</Code>.
+            <FormattedMessage
```

Comment:
@ryancreatescopy @wackerow FYI - here's an example I've added on how to incorporate code blocks within a sentence using variables.